discovery-provider/.env

```diff
@@ -8,4 +8,5 @@ audius_ipfs_port=6001
 audius_ipfs_gateway_hosts=
 audius_discprov_block_processing_window=100
 audius_discprov_peer_refresh_interval=10
+audius_discprov_identity_service_url=http://localhost:8000
```

Comment:
confirm this works with local env

Reply:
@hareeshnagaraj i know i said we should do this config side but i'm having second thoughts about this. we already define in the identity service in the libs. we could just pass this through and eliminate this variable